### ISSUE TITLE
Add a Face-Super Resolution module in the Awesome-Face-Operations folder 

### ISSUE THREAD

**Issue by Aarthy153** [akshitagupta15june/Face-X#1265]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of the approach to be followed.

**Additional context**

**Comment by akshitagupta15june** [comment]
can u describe what u want to implement

**Comment by Aarthy153** [comment]
Super- Resolution is the process of converting low-resolution images to high-resolution images. The approach is to provide super-resolution images with the concept of Deep learning. With this approach, you can perform super-resolution without affecting image quality. For this, the approach is named SRGAN, which is a generative adversarial network for single-image super-resolution. It uses a perceptual loss function which consists of an adversarial loss and a content loss. I am using pre-trained models which are EDSR and LAPSRN to implement this concept.
